src/PasswordStorage/ManagePassword.java

```diff
@@ -0,0 +1,118 @@
+package PasswordStorage;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+
+
+
+public class ManagePassword {
+
+    // Class will be used more later to separate writing new / managing existing passwords
+    //Skip below for working code
+
+    //private static String GetData(Scanner scanner){
+        //Scanner scanner = new Scanner(System.in);
+
+        // get credentials
+        //System.out.println("Enter a Title for the credentials:");
+        //System.out.println("Enter a Username");
+        //String username = scanner.nextLine();
+        //System.out.println("Enter a Password:");
+        //String password = scanner.nextLine();
+
+        //return scanner.nextLine();
+    
+    //}
+
+    // Used to write a new Password File at the moment - Append to come soon
+    // Add "ManagePassword.WritePassword();" to main to execute.
+    public static void WritePassword() {
+
+        try {
+            Scanner scanner = new Scanner(System.in);
+
+            // horrible way of getting data
+            System.out.println("Enter a Title for the credentials:");
+            String title = scanner.nextLine();
+            System.out.println("Enter a Username");
+            String username = scanner.nextLine();
+            System.out.println("Enter a Password:");
+            String password = scanner.nextLine();
+
```

Comment:
this should be moved to a testing file, the methods in this class should be designed to supply the methods to another class in the project.